homeassistant/components/home_connect/switch.py

```diff
@@ -180,6 +170,10 @@ async def async_update(self) -> None:
         )
 
 
+@deprecated_class(
+    "homeassistant.components.home_connect.select.HomeConnectProgramSelectEntity",
+    breaks_in_ha_version="2025.5",
+)
```

Comment:
That's not how we deprecate things though, I would recommend doing that in a followup. We want to do our due dilligence and use functions to see if the entities are used in automations or scripts, and if that is the case, raise an issue. 

For a good reference, this kind of deprecation once got added for `harmony` switch entities. #92787